### ISSUE TITLE
Auto complete

### ISSUE THREAD

**Issue by abku** [danhunsaker/angular-dynamic-forms#68]
Hi,

Has anyone attempted to add Auto complete to this mix.  Basically, when a user types any data into a text box I want to do a call to the server to get a list of matching records to get the user to pick one from.

Is this possible?

**Comment by danhunsaker** [comment]
Absolutely.  Just drop in whichever Angular autocomplete module you want to use, add the relevant attributes to the object in the `attributes` key of the field, and you should be set!

**Comment by abku** [comment]
@danhunsaker 
Is there a recommended one?
I have a few fields (not all input fields) and each have to query a different table in the database.
Eg:  Order Number field queries Order Table
       Item number queries Item table.... 

I like the simplest implementation as I am not that great yet with Angularjs...

**Comment by danhunsaker** [comment]
I tend to use Angular Bootstrap, myself.
